tests/threshold_tools/test_thresholds_returns.py

```diff
@@ -69,7 +70,7 @@ def test_return_period(T2_ams):
 
 # Test invalid distribution argument for Return Periods
 @pytest.mark.advanced
-def test_return_period_invalid_distr(T2_ams):
+def test_return_period_invalid_distr(T2_ams: xr.DataArray):
     with pytest.raises(ValueError, match="invalid distribution type"):
         rvs = threshold_tools.get_return_period(
```

Comment:
Unused variables like `rvs` should be replaced with `_` to indicate they aren't used.